docs/manifest.md

```diff
@@ -7,7 +7,7 @@ The manifest defines how a [dataset](data_model.md) is constructed.
 The following capabilities have been identified:
 
 | Capability           | Kind           | Motivation                                                |
-|----------------------|----------------|-----------------------------------------------------------|
+| -------------------- | -------------- | --------------------------------------------------------- |
```

Comment:
why this - here I see line goes to the end: https://www.markdownlang.com/extended/tables.html

I just approved and merged your MR to undo my earlier fix for this, read the change the wrong way 🙈

Anyway, pycharm complains and auto fixes to 
```
|----|---|
```
which is why i complained at this in the first place - you ide likes it like this?

Reply:
Ah interesting, my editor does this when requesting a format. I do not care really as long as we are consistent